src/pages/_app.tsx

```diff
@@ -99,6 +99,16 @@ const MyApp: React.FC<MyAppProps> = ({ Component, pageProps }) => {
     validateRoute(router, taxForm, taxFormUserInput, postponeUserInput)
   }, [router, nextRoute, taxForm, taxFormUserInput, postponeUserInput])
 
+  useEffect(() => {
+    const handleRouteChange = (url) => {
+      gtag('config', 'UA-69285708-14', { page_path: url })
```

Comment:
GMT ID je uz raz definovane v https://github.com/priznanie-digital/priznanie-digital/blob/master/src/pages/_document.tsx#L16, mozes to dat prosim ta niekam ako premennu? Napr `src/lib/gtm.ts`.

Reply:
Dorobil som to v #412